po/fa.po

```diff
@@ -1,122 +1,142 @@
 # Translators:
 # imni <iriman@chmail.ir>, 2018
-# 
+#
 msgid ""
 msgstr ""
 "Last-Translator: imni <iriman@chmail.ir>, 2018\n"
-"Language-Team: Persian (https://www.transifex.com/openstreetmap-france/teams/17462/fa/)\n"
+"Language-Team: Persian (https://www.transifex.com/openstreetmap-france/"
+"teams/17462/fa/)\n"
 "Language: fa\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 
-#: ../transport.validator.mapcss:5
 msgid "Uncompromising validation of transit data"
 msgstr ""
 
-#: ../transport.validator.mapcss:39 ../transport.validator.mapcss:44
 msgid "Missing transportation mode, add a tag route = bus/coach/tram/etc"
-msgstr ""
-"شیوهٔ حمل‌ونقل عمومی مشخص نشده. یکی از تگ‌های route = bus/coach/tram/...‎ را"
-" وارد کنید."
+msgstr "    .    route = bus/coach/tram/...   ."
```

Comment:
Translation lost

Reply:
the encoding was not set in the po, I don't know why :thinking: I've added it to fix this. Thanks for noticing !